mvn `shade` failure on project hazelcast-sql

- Fails on `Hazelcast-4.master-Solaris-OracleJDK8`
- Failures start on [Build #171 (Apr 25, 2020 7:18:02 PM)](http://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-4.master-Solaris-OracleJDK8/171/consoleFull)
- Failure can be traced in [console log](http://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-4.master-Solaris-OracleJDK8/175/consoleFull)
- Error 
```
02:42:27.552 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:3.2.2:shade (default) on project hazelcast-sql: Unable to parse configuration of mojo org.apache.maven.plugins:maven-shade-plugin:3.2.2:shade for parameter excludes: Cannot assign configuration entry 'excludes' with value 'com.hazelcast.**' of type java.lang.String to property of type java.util.List -> [Help 1]
```
kindly check

--- a/hazelcast-sql/pom.xml
+++ b/hazelcast-sql/pom.xml
@@ -97,7 +97,9 @@
                                 <relocation>
                                     <pattern>com.</pattern>
                                     <shadedPattern>${relocation.root}.com.</shadedPattern>
-                                    <excludes>com.hazelcast.**</excludes>
+                                    <excludes>
+                                        <exclude>com.hazelcast.**</exclude>
+                                    </excludes>
                                 </relocation>
                                 <relocation>
                                     <pattern>org.</pattern>

